### PR TITLE
Weather data script as of 9/4

### PR DESCRIPTION
**Pull request by stephpenn1**

### CODE REVIEW

File: R_scripts/weatherdat.R [hunk 1]
```diff
@@ -5,13 +5,75 @@ library(readr)
...
 library(ggrepel)
+library(lubridate)
+library(tidyr)
+
+# Function to replace S/N with descriptive label
+read_wxdat <- function(file) {
+  subfile <- read_csv("../inventory_data/wstation_info.csv")
+  wdat <- file %>% 
+    read_csv(skip=1) %>%
+    gather(label, Value, -1:-2) %>%
```
Comment (bpbond) [inline R_scripts/weatherdat.R]: A couple comments would be useful for the above lines, e.g. "Drop first two columns, which contain X and Y"

File: R_scripts/weatherdat.R [hunk 2]
```diff
@@ -5,13 +5,75 @@ library(readr)
...
+    gather(label, Value, -1:-2) %>% 
+    separate(label, sep = "SEN S/N: ", into = c("firstpart", "Sensor_SN")) %>%
+    separate(firstpart, sep = ",", into = c("Sensor_Group", "info")) %>%
+    select(- info)
+  wdat$Timestamp <- mdy_hms(wdat$`Date Time, GMT-04:00`)
+  wdat$Sensor_SN <- as.integer(gsub(")", "", wdat$Sensor_SN))
+ 
+  wdat <- left_join(wdat, subfile, by = "Sensor_SN") 
+  wdat <- separate(wdat, Sensor_Label, sep = "_", into = c("info1", "info2", "Sensor_Depth"))
+  wdat <- select(wdat, -`Date Time, GMT-04:00`, -info1, -info2)
```
Comment (bpbond) [inline R_scripts/weatherdat.R]: Same as above. What's happening here? A comment like "# Remove unneeded and 'junk' (from separate above) columns" would help.

File: R_scripts/weatherdat.R [hunk 3]
```diff
@@ -5,13 +5,75 @@ library(readr)
...
-  for (i in files) {
-    list[[i]] <- read_csv(i)
-  }
 }
 
-c <- read_weather_data("../weather_data/")
+cat("Reading weather data...")
+wx_LSLE <- read_wxdat("../weather_data/LSLE_weather_20180809.csv")
+wx_MSLE <- read_wxdat("../weather_data/MSLE_weather_20180809.csv")
+wx_HSLE <- read_wxdat("../weather_data/HSLE_weather_20180806.csv")
```
Comment (bpbond) [inline R_scripts/weatherdat.R]: Do you want these to be fixed filenames? What about future downloads?

Comment (stephpenn1) [reply R_scripts/weatherdat.R]: good idea, do you think I should switch it to something that finds the "LSLE" string, etc. and opens that file?

**Pull request merged**